src/shims/ApiCompatBaseline.netfx.netstandard20Only.txt

```diff
@@ -1,34 +1,12 @@
 Compat issues with assembly netstandard:
```

Comment:
Note this diff.  We're now down to a minimal set of known breaks between ns1.6 and net461.